src/clients/HydrogenClient.ts

```diff
@@ -106,6 +120,21 @@ class HydrogenClient {
       data: result.data.map(formatCrossChainTransferDetailed),
     }
   }
+
+  async getFeeQuote(req: GetFeeQuoteRequest): Promise<GetFeeQuoteResponse> {
+    this.checkState();
```

Comment:
This line needs to be commented out/removed as this throws an error if user is connected to Testnet/Localhost.